src/server/verification/verification.js

```diff
@@ -19,49 +20,50 @@ class Verifications implements VerificationAPI {
   /**
    * Verifies user
    * @param {UserRecord} user user details of the user going through FR
-   * @param {*} verificationData data from zoomsdk
+   * @param sessionId
+   * @param imagesBase64
+   * @param storage
 
    */
-  async verifyUser(user: UserRecord, verificationData: any) {
+  async verifyUser(user: UserRecord, sessionId: string, imagesBase64: any, storage: StorageAPI) {
     this.log.debug('Verifying user:', { user })
-    const sessionId = verificationData.sessionId
-    this.log.debug('sessionId:', { sessionId })
-    const searchData = Helper.prepareSearchData(verificationData)
-    // log.info('searchData', { searchData })
-    const isDuplicate = await Helper.isDuplicatesExist(searchData, verificationData.enrollmentIdentifier)
-    GunDBPublic.gun
-      .get(sessionId)
-      .get('isNotDuplicate')
-      .put(!isDuplicate) // publish to subscribers
 
-    this.log.debug('isDuplicate result:', { user: user.identifier, isDuplicate })
-    if (isDuplicate) return { ok: 1, isDuplicate }
-    const enrollData = Helper.prepareEnrollmentData(verificationData)
-    // log.info('enrollData', { enrollData })
-    const enrollResult: EnrollResult = await Helper.enroll(enrollData)
-    GunDBPublic.gun
-      .get(sessionId)
-      .get('isEnrolled')
-      .put(enrollResult.alreadyEnrolled || (enrollResult.enrollmentIdentifier ? true : false)) // publish to subscribers
-    const livenessFailed = (enrollResult && enrollResult.ok === false) || enrollResult.livenessResult === 'undetermined'
-    this.log.debug('liveness result:', { user: user.identifier, livenessFailed })
+    const enrollHandler = async (userId, sessionId, data) => {
+      this.log.debug('Verifying user enrollHandler :', data)
+      const checkFields = ['isDuplicate', 'isLive', 'isEnrolled']
 
-    GunDBPublic.gun
-      .get(sessionId)
-      .get('isLive')
-      .put(!livenessFailed) // publish to subscribers
-    if (livenessFailed) return { ok: 1, livenessPassed: false }
-
-    //this.log.debug('liveness result:', { user: user.identifier, livenessPassed }) // This is left to support future granularity for user better UX experience. Consider using authenticationFacemapIsLowQuality property https://dev.zoomlogin.com/zoomsdk/#/webservice-guide
-    //if (!livenessPassed) return { ok: 1, livenessPassed }
+      for (let field in data) {
```

Comment:
redundnat
gun.get(sessionId).put(result)
modify client to work with correct field names sent by server dont modify them here

Reply:
done in https://github.com/GoodDollar/GoodServer/pull/151/commits/46d0a0ef6b5ae651fa64ca55c721364da337d9d9

Reply:
you dont need to put each field separately
gun.get(sessionId).put(result) is enough

Reply:
```
import { pick } from 'lodash';
// ...
const enrollHandler = async (userId, sessionId, data) => {
  this.log.debug('Verifying user enrollHandler :', data)
  const enrollPayload = pick(data, 'isDuplicate', 'isLive', 'isEnrolled');

  GunDBPublic.gun
    .get(sessionId)
    .put(enrollPayload);
// ...
```